.github/workflows/docs-checks.yml

```diff
@@ -22,19 +22,18 @@ jobs:
         with:
           python-version: 3.9
 
-      - name: Weekly reset caching
-        run: echo "::set-output name=period::$(python -c 'import time ; days = time.time() / 60 / 60 / 24 ; print(int(days / 7))' 2>&1)"
-        id: times
+      - name: Reset caching
+        run: python -c "import time; days = time.time() / 60 / 60 / 24; print(f'TIME_PERIOD=d{int(days / 2) * 2}')" >> $GITHUB_ENV
 
       # Note: This uses an internal pip API and may not always work
       # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
       - name: Cache pip
         uses: actions/cache@v2
         with:
           path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-td${{ steps.times.outputs.period }}-${{ hashFiles('requirements.txt') }}
+          key: ${{ runner.os }}-pip-td${TIME_PERIOD}-${{ hashFiles('requirements.txt') }}
           restore-keys: |
-            ${{ runner.os }}-pip-td${{ steps.times.outputs.period }}-
+            ${{ runner.os }}-pip-td${TIME_PERIOD}-
```

Comment:
Shouldn't this be `{{ env.TIME_PERIOD }}` like the others?

Reply:
Nice catch! Thank you :) I'm throwing a quick PR now :]

Reply:
Updated in #12455.

Reply:
it does not have any effect but we can do it for consistency